### ISSUE TITLE
specify more than one teams

### ISSUE THREAD

**Issue by dszabo** [PeteGoo/tcSlackBuildNotifier#18]
It'd be great if I could specify user, token and team name project wide in teamcity (not just the channel), since we have multiple slack teams.

**Comment by PeteGoo** [comment]
I was wondering if this would become a scenario.

Project level could be doable.

**Comment by dszabo** [comment]
thanks for the comment! Btw I tried to build from sources with mvn package command, but I get maven dependency exceptions. Is the HEAD builds now correctly and something is just misconfigured on my side?

**Comment by PeteGoo** [comment]
Yeah, in master there is a a build failure but the web-ui project output should still contain the working zip package.

This is fixed in a feature branch I have.

**Comment by nafg** [comment]
+1

**Comment by Shayac** [comment]
+1

**Comment by PeteGoo** [comment]
I'm interested in more info on what this should look like.

For example, there could be a section on the project Slack tab....?

**Comment by nafg** [comment]
I think there should be a global screen to edit the list of teams and endpoints, and then each notification specification has to choose a team.

**Comment by PeteGoo** [comment]
Can you check out 1.4.3 and see if it solves your issue?

https://github.com/PeteGoo/tcSlackBuildNotifier/releases/tag/1.4.3

**Comment by gregopet** [comment]
Thank you for this feature, it was sorely missing :+1:
